codenteq/iyzico-payment-gateway
# Iyzico refund & bagisto_asset function fixed

## Issue Reference
bagisto_asset function fixed.

## Description
The customizable asset structure comes default from `bagisto_asset('images/money-transfer.png', 'shop');`. However, to customize, you can replace this code block with `bagisto_asset('images/iyzico.svg', 'iyzico');`.

> To load the assets from your package, add the following code to the config/bagisto-vite.php file:
```php
'iyzico' => [
     'hot_file'                 => 'iyzico-default-vite.hot',
     'build_directory'          => 'themes/iyzico/default/build',
     'package_assets_directory' => 'src/Resources/assets',
],
```
```bash 
npm run build || yarn build
```

## How To Test This?
To add a logo to the iyzico payment management in the admin console, navigate to admin/configuration/sales/payment_methods. The image resolution should be around 55px x 45px.

## Documentation
